### PR TITLE
feat(website): optimize  the import logic of demo.tsx

### PR DESCRIPTION
**Pull request by Daydreamer-riri**
Fix "the word "example" would appear first."

And I made some changes to the style

- before: 
image

- after
image

### CODE REVIEW

**Comment by childrentime** [comment]
Wait a second, I can't find the types in local.  So do in netlify.
```
npm run build:types
cd packages/website
npm run dev
```
Did it miss something?
image

**Comment by Daydreamer-riri** [comment]
Sorry, i will fix it

**Comment by Daydreamer-riri** [comment]
Sorry, i will fix it

File: packages/core/hooks/useEventListener/demo.tsx
Comment (childrentime) [inline packages/core/hooks/useEventListener/demo.tsx:1]: remove this in README Usage too, no more problems.

**Review by childrentime — APPROVED** [review]

**Pull request merged**